server/data/testData/ESCAPE_FROM_CUSTODY.ts

```diff
@@ -0,0 +1,6214 @@
+          nextQuestionId: '44415',
+        },
+        {
+          id: '182549',
+          code: 'OTHER',
+          active: true,
+          label: 'Other',
+          commentRequired: true,
+          dateRequired: false,
+          nextQuestionId: '44415',
+        },
+      ],
+    },
+    '45164': {
+      id: '45164',
+      active: true,
+      code: 'WERE ANY INJURIES SUSTAINED BY PRISONERS',
+      label: 'Were any injuries sustained by prisoners?',
+      multipleAnswers: false,
+      answers: [
+        {
+          id: '182606',
+          code: 'YES',
+          active: true,
+          label: 'Yes',
+          commentRequired: true,
+          dateRequired: false,
+          nextQuestionId: '44428',
+        },
+        {
+          id: '182607',
+          code: 'NO',
+          active: true,
+          label: 'No',
+          commentRequired: false,
+          dateRequired: false,
+          nextQuestionId: '44388',
+        },
+      ],
+    },
+    '45177': {
+      id: '45177',
+      active: true,
+      code: 'THE INCIDENT IS SUBJECT TO',
+      label: 'The incident is subject to',
+      multipleAnswers: true,
+      answers: [
+        {
+          id: '182648',
+          code: 'INVESTIGATION BY POLICE',
+          active: true,
+          label: 'Investigation by police',
+          commentRequired: false,
+          dateRequired: false,
+          nextQuestionId: '44798',
+        },
+        {
+          id: '182649',
+          code: 'INVESTIGATION INTERNALLY',
+          active: true,
+          label: 'Investigation internally',
+          commentRequired: false,
+          dateRequired: false,
+          nextQuestionId: '44798',
+        },
+        {
+          id: '182647',
+          code: "GOVERNOR'S ADJUDICATION",
+          active: true,
+          label: "Governor's adjudication",
+          commentRequired: false,
+          dateRequired: false,
+          nextQuestionId: '44798',
+        },
+        {
+          id: '182650',
+          code: 'NO INVESTIGATION',
+          active: true,
+          label: 'No investigation',
+          commentRequired: false,
+          dateRequired: false,
+          nextQuestionId: '44798',
+        },
+      ],
+    },
+  },
+  prisonerRoles: [
+    {
+      prisonerRole: 'ACTIVE_INVOLVEMENT',
+      onlyOneAllowed: false,
+      active: true,
+    },
+    {
+      prisonerRole: 'ASSISTED_STAFF',
+      onlyOneAllowed: false,
+      active: true,
+    },
+    {
+      prisonerRole: 'ESCAPE',
+      onlyOneAllowed: true,
+      active: true,
+    },
+    {
+      prisonerRole: 'IMPEDED_STAFF',
+      onlyOneAllowed: false,
+      active: true,
+    },
+    {
+      prisonerRole: 'PRESENT_AT_SCENE',
+      onlyOneAllowed: false,
+      active: true,
+    },
+    {
+      prisonerRole: 'SUSPECTED_INVOLVED',
+      onlyOneAllowed: false,
+      active: true,
+    },
+  ],
+} as const
+
+export const steps: FormWizard.Steps<FormWizard.MultiValues> = {
```

Comment:
FYI: I've also checked this whole expected `steps` dictionary manually yesterday against the SVG graph and how the grouping rules work [as you can imagine that is a lot of "fun" 😅] but feel free to triple check or spot check this if you wish